src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 101;
+    private Ball ball = new Ball();
```

Comment:
The 'ball' instance should not be a class-level field since it is only used within the 'getRandomBall' method. You can create a new 'Ball' instance inside the method each time it is called.